usaspending_api/common/threaded_data_loader.py

```diff
@@ -96,6 +96,9 @@ def load_from_file(self, filepath, encoding='utf-8'):
 
         row_queue.join()
 
+        for process in pool:
```

Comment:
To ensure all the processes close. If any of the processes hang or don't close for some reason, you can be left with some phantom processes that, while they aren't using local resources, are still maintaining a connection to the DB, which could cause you to fill max_connections with effectively ghosts.